pkg/sentry/kernel/task_start.go

```diff
@@ -243,6 +243,11 @@ func (ts *TaskSet) assignTIDsLocked(t *Task) error {
 		}
 		allocatedTIDs = append(allocatedTIDs, allocatedTID{ns, tid})
 	}
+	if t.tg.pidns.tgids[t.tg] == 1 {
```

Comment:
I think it would be better to set `inum` with `device.ProcDevice.NextIno()` when the namespace is created.